### ISSUE TITLE
modulesFolder not working

### ISSUE THREAD

**Issue by ahultgren** [fuse-box/fuse-box#124]
I want to shim node's `stream`, and from what I've understood `modulesFolder` is the way to go. Thus my `fuse.js` looks like this:

```
const fb = require("fuse-box");

const FuseBox = fb.FuseBox;

const fuse = FuseBox.init({
  homeDir: "source",
  sourceMap: {
    bundleReference: "public/dist/main.js.map",
    outFile: "public/dist/main.js.map",
  },
  outFile: "public/dist/main.js",
  modulesFolder: "source/shims"
});

fuse.devServer("> js/main.js");
```

my `source/shims/stream/index.js` contains:

```
module.exports = {
  test: "asd",
};
```

and my `source/js/main.js`:

```
console.log("test", require("stream"));
```

## Expected result

To log `"test", {test: "asd"}`.

## Actual result

`"test", {}`.

Am I doing something wrong?

**Comment by nchanged** [comment]
Have you disabled cache? Because your library will treated like any other NPM module.

Btw, We have `stream` library already in [place](https://github.com/fuse-box/fuse-box/tree/master/assets/libs/stream)

**Comment by ahultgren** [comment]
If by disabling cache you mean setting `cache: false` and/or rm-rf:ing the .fusebox folder then yes.

It seems to me the provided stream library doesn't [do much](https://github.com/fuse-box/fuse-box/blob/master/assets/libs/stream/index.js)?

**Comment by nchanged** [comment]
Well, not for browser yes. Only server. I will check, maybe there is an issue with custom module priorities

**Comment by nchanged** [comment]
Fixed in `v1.3.111`
